CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+# 0.47.0 / 2022-09-13
+
+* [FEATURE] Add ZGC Cycles and ZGC Pauses beans support out of the box [#393][]
+* [FEATURE] Splits the JCommander code into it's own class to allow embedded applications to exclude it [#397][]
+* [SECURITY] Bump `snakeyaml` dependency to version `1.31` to mitigate [CVE-2022-25857][] [#398][]
```

Comment:
nit: for consistency with the rest of the CHANGELOG should we directly link the CVE here instead of using an indirect link?